app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposer.kt

```diff
@@ -400,17 +399,16 @@ private fun MessageComposeActions(messageComposerState: MessageComposerInnerStat
 }
 
 @Composable
-private fun AdditionalOptionButton(attachmentOptionsInitialState: Boolean = false, onClick: () -> Unit) {
-    val rotationAngle by animateAsStateRotationToRight(isOpen = attachmentOptionsInitialState)
+private fun AdditionalOptionButton(isOpen: Boolean = false, onClick: () -> Unit) {
```

Comment:
to me the name suggest that the button can be open or closed, but in fact is is setting the state to selected or default, so maybe instead of isOpen, we could find some better naming ? maybe isSelected would be enough, what do you think ?

Reply:
Totally agree!